packages/loopring_v3/test/testBlockVerifier.ts

```diff
@@ -560,11 +576,33 @@ contract("BlockVerifier", (accounts: string[]) => {
         onchainDataAvailability,
         commitBlocksSize1[0].blockSize,
         commitBlocksSize1[0].blockVersion,
-        publicDataHashes,
+        publicInputs,
         exchangeTestUtil.flattenList(commitBlocksSize1.map(x => x.proof)),
         { from: anyone }
       );
       assert(!success, "verification should not be succesful");
     });
+
+    it("should not be able to verify multiple blocks of the same circuits with public input > scalar field", async () => {
```

Comment:
In the current implementation, public input == scalar field will also fail.

Reply:
Yes, it should say `>=`, will fix.